iceoryx_examples/iceperf/CMakeLists.txt

```diff
@@ -1,15 +1,30 @@
+# Copyright (c) 2020 by Robert Bosch GmbH, Apex.AI Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 # Build iceperf example
 cmake_minimum_required(VERSION 3.5)
 project(example_iceperf)
 
 find_package(iceoryx_posh CONFIG REQUIRED)
+find_package(iceoryx_binding_c CONFIG REQUIRED)
```

Comment:
Is it then possible to build the examples without an activated C binding?

Reply:
No.

Reply:
will be addressed in #452